test-app/config/ember-try.js

```diff
@@ -27,7 +27,7 @@ module.exports = async function () {
         name: 'ember-lts-3.28',
         npm: {
           devDependencies: {
-            'ember-source': '~3.28.0',
+            'ember-source': '~3.28.8',
```

Comment:
Note that while this is fine, it's not actually necessary: `~` here will select the most recently *patch* version available, so it'll automatically and always get the latest here. If we're changing it explicitly, let's bump it to the latest: `3.28.9`.

Reply:
@chriskrycho agree, this indeed unnecessary. for some reason I tried to do what package manager does on it's own.
Anyway bumped to `3.28.9` for clarity as requested